src/content/docs/tutorial-kubernetes-learn/tutorial-k8s-layers.mdx

```diff
@@ -0,0 +1,185 @@
+
+* Red or yellow tiles: think of yellow tiles as warnings to keep an eye on what it monitors. For example, if you have 2 unhealthy deployments you should take note and possibly plan to troubleshoot those deployments. Think of red tiles as critical alerts. These aren't necessarily failures in your system, but you should prioritize addressing them as immediately as possible.  
+* Anomalous spikes in graphs: there are various graphs that show things such as pending pods over time or even memory utilization over time. Spike are fairly normal, such as the spikes in the **Kubernetes Warning Events by Reason** graph in the screenshot above. These spikes happen regularly about every 5 minutes so they are no cause for concern. Look for spikes that happen outside of regular patterns or spike in a much larger magnitude than normal. 
+* Node readiness: observe whether nodes in the cluster are ready and able to host pods. Ensure that your cluster's infrastructure is prepared to handle workloads without any bottlenecks.
+* Resource count insights: keep a close eye on the number of pods, containers, nodes, or other Kubernetes resources within the cluster. While you won't always find something actionable, monitoring resource utilization allows you to plan for future scaling.
+
+Use the time selector in the top left of the page to see your data across time ranges to verify any troubling data isn't just random or to triage across a longer timeframe. 
+    </Step>
+</Steps>
+
+## Understand and monitor the orchestrated layer
+
+Continuing with our bakery analogy, the orchestrated layer is the baking stations within the bakery. Each station houses a set of ovens that work together to fulfill specific tasks. Each station might be designated to have three ovens running at all times. If an oven fails, the station will fire up a new oven to keep up with expectations.
+
+In a Kubernetes system, deployments have various specifications. For example, a deplyoment might be specified to always run four instances of an application in individual containers. The deployment will spin up pods containing that application until it meets that quota. If a pod were to fail, it would spin up a new one to continue to meet the specified  .
+
+
+The following steps guide you through a general strategy to monitor your deployments:
+
+<Steps>
+    <Step>
+### Navigate to the overview dashboard
+
+Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Kubernetes > Overview Dashboard**.         
+    </Step>
+    <Step>
+### Triage your deployments
+
+Take a look at the unhealthy deployment tile and the deployments tile. Compare the two to measure the percentage of unhealthy deployments in your cluster. Unhealthy deployments are deployments with missing or unavailable pods. This usually means that the deployment was unable to spin up those pods. Let's take a close look.
```

Comment:
Two uses of deployments here, I'd just say something like "Unhealthy deployments have"